### ISSUE TITLE
Replace "steps" by "epochs"

### ISSUE THREAD

**Issue by jondo** [tensorflow/tensorboard#5806]
## Environment information (required)

TensorBoard 2.9.1

## Issue description

In Scalars and Time Series dashboards, the horizontal axes of TensorBoard diagrams can be configured to "Steps", meaning epochs. This is confusing, because at other places in TensorFlow "steps" means batches, see e.g. in the [documentation of validation_steps](https://keras.io/api/models/model_training_apis/#fit-method):
> steps (batches of samples)

I suggest replacing "steps" by "epochs" in TensorBoard.

**Comment by bileschi** [comment]
The semantic meaning of steps, from the perspective of TensorBoard, is up to the user.  It *may* mean epochs, or it may mean gradient steps, or it may mean some other user-defined strictly increasing value.  Epochs has a more established meaning in ML.  

Thank you for taking the time to make the request, but we are unlikely to make this change.

**Comment by jondo** [comment]
I understand. Thank you for this explanation, which can also help other people finding this issue.
